bin/zopen-clean

```diff
@@ -111,10 +117,21 @@ cleanDangling()
 
 cleanCache()
 {
-  printVerbose "Cleaning ${ZOPEN_ROOTFS}/var/cache/zopen"
-  rm -rf "${ZOPEN_ROOTFS}/var/cache/zopen/"*
-  syslog "${ZOPEN_LOG_PATH}/audit.log" "${LOG_A}" "${CAT_FILE}" "CLEAN" "cleanCache" "Main cache in ${ZOPEN_ROOTFS}/var/cache/zopen cleaned"
-  printInfo "- Cache at '${ZOPEN_ROOTFS}/var/cache/zopen' cleaned"
+  # "All" option ignores the input parameter list as it brute force-cleans the cache rather
+  # than cherry-pick the files to remove
+  if ${all}; then
+    printVerbose "Cleaning all entries in ${ZOPEN_ROOTFS}/var/cache/zopen"
+    rm -rf "${ZOPEN_ROOTFS}/var/cache/zopen/"*
+    syslog "${ZOPEN_LOG_PATH}/audit.log" "${LOG_A}" "${CAT_FILE}" "CLEAN" "cleanCache" "Main cache in ${ZOPEN_ROOTFS}/var/cache/zopen cleaned"
+    printInfo "- Cache at '${ZOPEN_ROOTFS}/var/cache/zopen' cleaned"
+  else
+    for needle in $1; do
+      printVerbose "Cleaning ${ZOPEN_ROOTFS}/var/cache/zopen entries for '${needle}"
+      rm -rf "${ZOPEN_ROOTFS}"/var/cache/zopen/"${needle}"-*
+      syslog "${ZOPEN_LOG_PATH}/audit.log" "${LOG_A}" "${CAT_FILE}" "CLEAN" "cleanCache" "Cache for '${needle}' in ${ZOPEN_ROOTFS}/var/cache/zopen cleaned."
+      printVerbose "- Cleaned cached files for '${needle}' in cache at '${ZOPEN_ROOTFS}/var/cache/zopen'."
```

Comment:
Should we add an option to clean the json cache? 
zopen_releases.json
zopen_releases.timestamp
zopen_releases.timestamp.current